src/datums.jl

```diff
@@ -498,15 +498,15 @@ abstract type RGF93v2b <: Datum end
 ellipsoid(::Type{RGF93v2b}) = GRS80🌎
 
 """
-    RNB72
+    BD72
```

Comment:
Are you using a name that is similar to the one used by C PROJ? We also need to make sure that the struct is defined in the file in alphabetical order.

Reply:
Yes BD72 is the alias for [this datum](https://epsg.org/datum_6313/Reseau-National-Belge-1972.html) and it is the one used in PROJ